pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[project]
+name = "pems"
+version = "0.0.1"
+description = "Caltrans Performance Measurement System (PeMS) is an application that enables access to traffic data collected by sensors that span the freeway system across all major metropolitan areas of the State of California."
+readme = "README.md"
+license = { file = "LICENSE" }
+classifiers = ["Programming Language :: Python :: 3 :: Only"]
+requires-python = ">=3.12"
+maintainers = [
+  { name = "Compiler LLC", email = "dev@compiler.la" }
+]
+dependencies = [
+]
+
+[project.optional-dependencies]
+dev = [
+    "black",
+    "flake8",
+    "pre-commit"
+]
+
+[project.urls]
+Code = "https://github.com/compilerla/pems"
+Homepage = "https://pems.dot.ca.gov/"
+Issues = "https://github.com/compilerla/pems/issues"
```

Comment:
NBD, but we could link to our `Docs` page here

Reply:
Ah, definitely, updated in 1ec5dcfaec9e4aeb5c78635bbb611852187103d7

Reply:
Ah, I meant we could add it! But this is fine, we can add the link for the new site once we have it up.